crates/uv/tests/lock_scenarios.rs

```diff
@@ -727,7 +727,7 @@ fn fork_non_local_fork_marker_direct() -> Result<()> {
     ----- stderr -----
     warning: `uv lock` is experimental and may change without warning.
       × No solution found when resolving dependencies:
-      ╰─▶ Because package-b{sys_platform == 'darwin'}==1.0.0 depends on package-c>=2.0.0 and package-a{sys_platform == 'linux'}==1.0.0 depends on package-c<2.0.0, we can conclude that package-a{sys_platform == 'linux'}==1.0.0 and package-b{sys_platform == 'darwin'}==1.0.0 are incompatible.
+      ╰─▶ Because package-b==1.0.0 depends on package-c>=2.0.0 and package-a==1.0.0 depends on package-c<2.0.0, we can conclude that package-a==1.0.0 and package-b{sys_platform == 'darwin'}==1.0.0 are incompatible.
```

Comment:
There is still some weirdness here -- we might want to throw out the markers entirely when reporting? I know how to do that.